python/lsst/sims/photUtils/applyIGM.py

```diff
@@ -35,24 +36,25 @@ def initializeIGM(self, zMin = 1.5, zMax = 2.9, zDelta = 0.1, minWavelen = 300):
         self.zMin = zMin
         self.zMax = zMax
         self.zDelta = zDelta
-        self.minWavelen = minWavelen 
+        self.minWavelen = minWavelen
         #Don't have max wavelength since transmission goes to 1.0 at longest wavelengths
         self.zRange = np.arange(zMin, zMax + (zDelta/2.), zDelta)
 
         if self.tablesPresent == False:
-            self.loadTables(str(str(os.environ['SIMS_SED_LIBRARY_DIR']) + '/igm/'))
+            self.loadTables(str(str(getPackageDir('sims_photUtils')) +
+                            '/python/lsst/sims/photUtils/igm_tables'))
```

Comment:
Does this not work if you use os.path.join(getPackageDir('sims_photUtils'), 'python/lsst/sims/photUtils/igm_tables') ?